src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -56,13 +58,22 @@ class ContentExplorer extends Component {
          * @param {Array} newFoldersPath
          */
         onEnterFolder: PropTypes.func.isRequired,
+        /** Called when the folders path is updated */
+        onFoldersPath: PropTypes.func,
+        /** Called when the include subfolders toggle is changed */
+        onIncludeSubfoldersToggle: PropTypes.func,
         /**
          * Called when an item is selected
          *
          * @param {Object} selectedItem
          * @param {number} selectedItemIndex
          */
         onSelectItem: PropTypes.func,
+        /** Called when an item is selected
```

Comment:
this is called whenever the selected items list changes. not just when one single one is selected